examples/tests/sdk_tests/resource/resource.test.w

```diff
@@ -0,0 +1,77 @@
+bring sim;
+
+inflight class CounterBackend impl sim.IResource {
```

Comment:
`sim.IResource` is slightly confusing because it's overloaded with `std.IResource` which I *guess* means something else. Maybe not?!

I am okay leaving it like this actually.

Reply:
I agree (I'm not sure what's better)

An alternative could be sim.IBackend

Reply:
Let's leave it with "resource". I think it's probably the same thing... :-)